crates/kitsune_p2p/fetch/src/rough_sized.rs

```diff
@@ -92,6 +95,13 @@ impl<T: Clone> RoughSized<T> {
     }
 }
 
+impl<T: Clone> RoughSized<T> {
+    /// Accessor
+    pub fn data(&self) -> T {
+        self.data.clone()
+    }
+}
+
```

Comment:
You can also access data through Deref. I think a lot of places you use `.data()` you could also just use `.clone()` and it would be more explicit. But this is fine too, since we only use this with Arc'd hashes anyway. But if we were going to use this more generally, I would push a bit harder.